Update quarantine-policies.md (#10172)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/quarantine-policies.md

   :::image type="content" source="../../media/mdo-quarantine-policy-quarantine-notification-settings.png" alt-text="Quarantine notification settings flyout in the Microsoft 365 Defender portal." lightbox="../../media/mdo-quarantine-policy-quarantine-notification-settings.png":::

> please note that we don't allow the same display name, the same subject, or the same disclaimer for different languages. so, the customer needs to provide different display name / subject / disclaimer for different languages selected

Reviewer: ```suggestion
Please note that we don't allow the same display name, the same subject, or the same disclaimer for different languages. You should provide a different display name, subject, and disclaimer for each language selected.
```
